meta/main.yml

```diff
@@ -22,14 +22,17 @@ galaxy_info:
   - failover
 dependencies:
   - role: brianshumate.consul
-    when: patroni_dcs == "consul"
+    when: patroni_dcs == "consul" and
+          not patroni_dcs_exist

```

Comment:
We're almost there. The variable name `patroni_dcs_exists` declared in `defaults/main.yml` must exactly match the variable name used in `meta/main.yml`